fineract-provider/src/main/java/org/apache/fineract/portfolio/calendar/service/CalendarUtils.java

```diff
@@ -114,7 +116,7 @@ private static Date convertToiCal4JCompatibleDate(final LocalDate inputDate) {
         try {
             formattedDate = new Date(seedDateStr, "yyyy-MM-dd");
         } catch (final ParseException e) {
-            e.printStackTrace();
+            LOG.error("{}",e.getMessage());
```

Comment:
Two thoughts on this this: a) your LOG.error() is wrong, as everywhere else, please:

```suggestion
            LOG.error("Invalid date: " + seedDateStr, e);
```

but, b) more importantly, isn't this catch plain wrong, not required, and can just be removed?! `new Date()` does not throw ParseException - or am I missing something obvious?

Reply:
It does throw a parse exception, a possible explanation is the string received might be complete gibberish and could not be parsed as a date?
![image](https://user-images.githubusercontent.com/42006277/82574789-3fc4e980-9ba5-11ea-8822-11df0633975a.png)

Reply:
Replacing it with:
LOG.error("Invalid date: {} ", seedDateStr, e);
or spot bugs won't like it :)

Reply:
LGTM!